aliases/available/general.aliases.bash

```diff
@@ -34,6 +34,15 @@ alias irc="$IRC_CLIENT"
 
 alias rb='ruby'
 
+# find Space hogs
+alias ducks='du -cks * | sort -nr | head -11 |cut -f2- | xargs du -hs'
+
+# Set VI Mode
+alias sovi='set -o vi'
+
+# Sudo to root keeping your user environment
+alias rsu='sudo -E su -p'
+
```

Comment:
`rsu` can be defined as:

`sudo -E -s`

I think, no need to use sudo an su to achieve this, but not sure if this works in OS X.